stdlib/public/core/Reverse.swift

```diff
@@ -523,22 +314,6 @@ extension LazyCollectionProtocol
   }
 }
 
-extension LazyCollectionProtocol
-  where
-  Self : RandomAccessCollection,
-  Elements : RandomAccessCollection {
-
-  /// Returns the elements of the collection in reverse order.
-  ///
-  /// - Complexity: O(1)
-  @_inlineable
-  public func reversed() -> LazyRandomAccessCollection<
```

Comment:
Whoops, we still need this, up until the point where the Lazy collections also use conditional conformances